travis/no-make-error.sh

```diff
@@ -1,19 +1,19 @@
-#!/bin/bash -x
+#!/bin/bash
 
 cd $1
 shift
 
-make $* > /dev/null 2> make-err
+echo "Running make $*"
+make $* 2>&1 > make-out
 
-RESULT=$(cat make-err)
+RESULT=$?
 
-rm make-err -f
-
-if [ "${RESULT}" != "" ]
+if [ "${RESULT}" != "0" ]
 then
-    echo There is some error output in the make
-    make $* > /dev/null
-    cd - > /dev/null
-    exit 1
+    echo There was errors in the make:
+    cat make-out
+    rm make-out
+    exit ${RESULT}
+else
+    rm make-out
 fi
-cd - > /dev/null
```

Comment:
All this change is wrong, this script is or avoid something like:
```make
TEST = $(shell ls unexisting)
```
witch return something in the error log but no error code ...